contracts/bridge/BridgeTransferCommon.sol

```diff
@@ -12,10 +12,10 @@ contract BridgeTransfer is BridgeHandledRequests, BridgeFee, BridgeOperator {
     bool public isRunning;
 
     uint64 public requestNonce; // the number of value transfer request that this contract received.
-    uint64 public minUnhandledRequestNonce; // a minimum nonce of a value transfer request that will be handled.
-    uint64 public maxHandledRequestedNonce; // maximum nonce of the counterpart bridge's value transfer request that is handled.
+    uint64 public lowerHandleNonce; // the nonce + 1 of a value transfer request surely processed by the counterpart bridge.
```

Comment:
How about keep the comment?
`// a minimum nonce of a value transfer request that will be handled.`